tabs/src/components/code/tab/deepLink/DeepLinkCode.jsx

```diff
@@ -0,0 +1,28 @@
+import * as microsoftTeams from '@microsoft/teams-js';
+import { Button, Text } from '@fluentui/react-northstar';
+
+function CreateNewMeeting() {
+  microsoftTeams.initialize();
+  const url = 'https://teams.microsoft.com/l/meeting/new?subject=New%20meeting%20&content=DeepLink%3A%20New%20meeting';
+  microsoftTeams.executeDeepLink(url);
+}
+
+function MoveToTab() {
+  microsoftTeams.initialize();
+  const appId = 'd68b7709-73bf-4add-a370-e89c78d7224e';
```

Comment:
Is this value fixed?
can be ref from default.env.json file? or an constance value

Reply:
No, it's a local test value. I will put it into a env.default.json file. Thanks.